lists/DolmaShoppingList.md

```diff
@@ -1,4 +1,9 @@
-Potatoes
-Leak
-Cheese
-Flowers
+Potatoes 
+Leak 
+Cheese 
+Flowers 
+Chocolate
+Lamb
+Rosemary
+Wine
```

Comment:
Again, which box? Which colour?

Reply:
resolved in https://github.com/pbuttigieg/shoppingList/pull/17/commits/25c7b3cbce8c47765a802b9176680890db228a59